packages/sdk/src/index.ts

```diff
@@ -1,18 +1,16 @@
 export * from './execute';
 export * from './constants';
+export * from './burn/burn';
+export * from './addMinter/addMinter';
+export * from './removeMinter/removeMinter';
 export * from './transfer/transfer';
-// export * from './burn/burn';
-// export * from './addMinter/addMinter';
-// export * from './removeMinter/removeMinter';
-// export * from './batchChangeMinters/batchChangeMinters';
-// export * from './deployContract/deployContract';
-// export * from './transferContractOwnership/transferContractOwnership';
-// export * from './list/list';
-// export * from './buy/buy';
-// export * from './depositStorage/depositStorage';
-// export * from './mint/mint';
-// export * from './revoke/revoke';
-export * from './v1/token';
-export * from './v1/token.types';
-export * from './v1/market';
-export * from './v1/market.types';
+export * from './batchChangeMinters/batchChangeMinters';
+export * from './deployContract/deployContract';
+export * from './transferContractOwnership/transferContractOwnership';
+export * from './list/list';
+export * from './buy/buy';
+export * from './depositStorage/depositStorage';
+export * from './mint/mint';
+export * from './revoke/revoke';
+
+
```

Comment:
Don't you need to `export * from './v1'` here to export both? Should we namespace it?

Reply:
Long story but typescript makes it super hard
if we did `export * from './v1'` we would need to do 
`import {v1} from '@mintbase-js/sdk'`
`const {mint} = v1`

I just added a index js to the v1 folder and now we can do 

`import {v1} from '@mintbase-js/sdk/lib/v1'`

Reply:
Also it seems that on omnisite we aren't even using that many wrappers to the point where only 1 or 2 changes were needed and we wont even need to import v1 there at all